lib/laa_crime_schemas/structs/means_details.rb

```diff
@@ -0,0 +1,88 @@
+# frozen_string_literal: true
+
+module LaaCrimeSchemas
+  module Structs
+    class MeansDetails < Base
+      attribute :employment_status, Types::EmploymentStatusType
+
+      attribute :income_details do # rubocop:disable Metrics/BlockLength
+        attribute :benefits, Types::Array.of(Base) do
+          attribute :type, Types::OtherBenefitType
```

Comment:
Whats the reason for 'Other' in the type name here? Is this based on the way it is named in the UI

Reply:
These are a subset of benefits that are combined to produce the single value "other_benefits_amount" used by MAAT.